witness/src/Dscp/Witness/Logic/Getters.hs

```diff
@@ -179,6 +179,11 @@ getTxMaybe gTxId = runMaybeT $ asum
     , fmap (GPublicationTxWitnessed . piTw) . MaybeT $
           SD.queryOne (coerce @_ @PublicationTxId gTxId)
     ]
+  where
```

Comment:
What is this for?
Seems to me that it is incomplete and used nowhere, were implementing something and then forgot about it?

Apart from this everything seems fine to me

Reply:
I was seeking a way to write this function so that, when new transaction types are added, compiler reminds to update this function as well. Otherwise, we will get a subtle bug.
And I came to this )

Reply:
Oh ok, I get it now.
I have nothing else to comment on.